### PR TITLE
Updated terminology and messaging throughout

### PR DESCRIPTION
**Pull request by nateziemann**

### CODE REVIEW

File: pipelines.adoc [hunk 1]
```diff
@@ -4,20 +4,20 @@ permalink: /guides/pipelines-getting-started/
...
+Kabanero uses link:https://github.com/tektoncd/pipeline/tree/master/docs#usage[Tekton Pipelines] to illustrate a continuous input and continuous delivery (CI/CD) workflow. Each of the featured link:https://github.com/kabanero-io/collections[ Collections] contains a default pipeline that builds the collection, publishes the image to a container registry, and then deploys the application to the Kubernetes cluster. You can also create your own tasks and pipelines and customize the pre-built pipelines and tasks.   All tasks and pipelines are activated by  link:https://github.com/kabanero-io/kabanero-operator[Kabanero's standard Kubernetes Operator].
```
Comment (dmuelle) [inline pipelines.adoc:12]: remove extra space between "tasks." and "All tasks and pipelines...."

`the pre-built pipelines and tasks.   All tasks and pipelines are activated by`

File: pipelines.adoc [hunk 2]
```diff
@@ -4,20 +4,20 @@ permalink: /guides/pipelines-getting-started/
...
 
-If you are building a new collection, the default tasks and pipelines are automatically pulled into your collections repo. If you want to customize the default tasks or pipelines, you can choose to apply your changes either to all collections or to a specific collection.  To apply your changes to all the collections,  update the files in the `incubator/common/pipelines/default` directory in your collections repo. If you want to update tasks or pipelines in a particular collection, make your changes in the pipelines directory under that collection in your collections repo.
+If you are building a new collection, the default tasks and pipelines are automatically pulled into your collections repo. If you want to customize the default tasks or pipelines, you can choose to apply your changes either to all Application Stacks or to a specific Application Stack.  To apply your changes to all,  update the files in the `incubator/common/pipelines/default` directory in your collections repo. If you want to update tasks or pipelines for one, make your changes in the pipelines directory under that collection in your collections repo.
```
Comment (dmuelle) [inline pipelines.adoc:20]: should `Application Stack` be capitalized in this context? IBM terminology lists it as lower case.

Remove double spaces after periods

"or to a specific Application Stack.  To apply your changes to all,  update the files in the"

File: pipelines.adoc [hunk 3]
```diff
@@ -39,25 +39,25 @@ By default, the pipelines run and deploy the application in the `+kabanero+` nam
...
 
 === Getting started
 
 ==== Set up a persistent volume to run pipelines
 
-Tekton pipelines require a configured volume that is used by the framework to share data across tasks. The `build task`, which uses `Buildah`, also requires a volume mount. The pipelinerun creates a Persistent Volume Claim (PVC) with a requirement for five Gi of persistent volume.
+Pipelines require a configured volume that is used by the framework to share data across tasks. The `build task`, which uses `Buildah`, also requires a volume mount. The pipelinerun creates a Persistent Volume Claim (PVC) with a requirement for five Gi of persistent volume.
 
-. Log in to your cluster. For example, for OKD
+. Log in to your cluster. For example,
```
Comment (dmuelle) [inline pipelines.adoc:60]: Is the example still correct as shown if it is not applied to OKD?

these prompts should be rewritten as follows-

Log in to your cluster, as shown in the following example

File: pipelines.adoc [hunk 4]
```diff
@@ -94,7 +94,7 @@ You can use the link:https://github.com/tektoncd/experimental/blob/master/webhoo
 
 If you are developing a new pipeline and want to test it in a tight loop, you might want to use a script or manually drive the pipeline.
 
-. Log in to your cluster. For example for OKD,
+. Log in to your cluster. For example,
```
Comment (dmuelle) [inline pipelines.adoc:97]: see previous comment

File: pipelines.adoc [hunk 5]
```diff
@@ -127,7 +127,7 @@ cd ./pipelines/incubator/manual-pipeline-runs/
 
 === Run pipelines manually from the command line
 
-. Login to your cluster. For example for OKD,
+. Login to your cluster. For example,
```
Comment (dmuelle) [inline pipelines.adoc:130]: see previous comment

File: pipelines.adoc [hunk 6]
```diff
@@ -39,25 +39,25 @@ By default, the pipelines run and deploy the application in the `+kabanero+` nam
...
-== Running Kabanero pipelines
+== Running Pipelines
 
-Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
+Explore how to use Pipelines to build and manage Collections.
 
 === Prerequisites
 
-. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.
+. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.  At present Kabanero 0.3.0 and greater only supports OCP 4.2+.  When OKD provides a comparable 4.x stream, Kabanero will again support OKD providing a full open source stack.
```
Comment (dmuelle) [inline pipelines.adoc:48]: edit to remove future tense and double space

**change**
cluster.  At present Kabanero 0.3.0 and greater only supports OCP 4.2+.  When OKD provides a comparable 4.x stream, Kabanero will again support OKD providing a full open source stack.

**to**
cluster. Presently, Kabanero 0.3.0 and greater supports only OCP 4.2+. When OKD provides a comparable 4.x stream, Kabanero is projected to again support OKD, providing a full open source stack.

**Review by dmuelle — COMMENTED** [review]
Hi Nate- see below for recommended minor edits to remove double spacing and adhere to style guidelines. The terminology updates all look good. If it's easier for me to make the other edits in a separate PR just let me know. Thanks

File: pipelines.adoc (hunk 1 above)
Comment (dmuelle) [inline pipelines.adoc:12]: Should `Tekton Pipelines` be just `Pipelines` ?

File: pipelines.adoc (hunk 2 above)
Comment (nateziemann) [reply pipelines.adoc:20]: Application Stacks is a feature. It should be capitalized. Other spacing and format changes look fine. Let’s get this PR merged pls. Thnx.

File: pipelines.adoc (hunk 1 above)
Comment (nateziemann) [reply pipelines.adoc:12]: Pipelines

File: pipelines.adoc (hunk 4 above)
Comment (nateziemann) [reply pipelines.adoc:97]: It’s fine as an example for now. Soon we’ll be back to OKD.

**Review by nateziemann — COMMENTED** [review]
Application Stacks is a feature. It should be capitalized. Other spacing and format changes look fine. Let’s get this PR merged pls. Thnx.

**Review by dmuelle — APPROVED** [review]

**Comment by dmuelle** [comment]
files is updated- merging changes

**Pull request merged**